js/services/vCardProperties.js

```diff
@@ -124,14 +127,17 @@ angular.module('contactsApp')
 		'X-SOCIALPROFILE': {
 			multiple: true,
 			readableName: t('contacts', 'Social network'),
-			template: 'text',
+			template: 'username',
 			defaultValue: {
 				value:[''],
 				meta:{type:['facebook']}
 			},
 			options: [
 				{id: 'FACEBOOK', name: 'Facebook'},
+				{id: 'INSTAGRAM', name: 'Instagram'},
+				{id: 'LINKEDIN', name: 'LinkedIn'},
 				{id: 'TWITTER', name: 'Twitter'}
+
```

Comment:
As said in the other comment let’s first start with the basics, not all and every single service we can think of. :) We can always extend later, but let’s not make this pull request too big.